security/comms/typha-node-tls.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Provide TLS certificates for Typha and Node
+description: Add TLS certificates to secure communications between if you are using Typha to scale your deployment.
+---
+
+### Big picture
+
+Provide TLS certificates that allow mutual TLS authentication between Node and Typha.
```

Comment:
Since operator isn't the "main" installation path we should include a note that says this only applies for the Operator or OpenShift installations.